src/components/organisms/NavBarSchedule/NavBarSchedule.scss

```diff
@@ -12,18 +11,17 @@ $weekday--margin-right: 10px;
   z-index: z(navbar-schedule);
   top: 0;
   left: 0;
-  width: 100%;
-  height: $NavBarSchedule--height;
+  height: 100%;
+  width: 40%;
 
   padding-top: $navbar-height;
   padding-left: $NavBarSchedule--padding;
 
-  background-color: opaque-black(0.9);
+  background-color: opaque-black(0.8);
   overflow: auto;
 
   pointer-events: none;
   opacity: 0;
-  transform: translateY(-$NavBarSchedule--height);
```

Comment:
instead of just removing `transform` property I'd make the schedule appear from the left: `translateX(-$NavBarSchedule--width)`

Reply:
fixed in 33957fc21429a0fe972b60cc5123bd1381ee81f5